di-ipv-cimit-stub/lambdas/put-contra-indicators/src/main/java/uk/gov/di/ipv/core/putcontraindicators/dto/CredentialSubject.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.di.ipv.core.putcontraindicators.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record CredentialSubject(List<DrivingPermit> drivingPermit, List<Passport> passport) {
+    public CredentialSubject(List<DrivingPermit> drivingPermit, List<Passport> passport) {
+        this.drivingPermit = drivingPermit == null ? List.of() : drivingPermit;
+        this.passport = passport == null ? List.of() : passport;
+    }
```

Comment:
Yep - it's part of the VcClaim. The document is included in the credential subject part of the VC, rather than the evidence.

Reply:
I meant this constructor, rather than the class, but happy if it's helpful to provide a default (just noting that it doesn't match the structure of the JSON)!

Reply:
...and, it's gone.